api/MediaMetadata.json

```diff
@@ -5,7 +5,7 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaMetadata",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": true
```

Comment:
```suggestion
            "version_added": "57"
```

Reply:
Please add this number to WebView also. Please do so for every place I change the Chrome value.